Quarkus cli test failing

**Describe the bug**

The `io.quarkus.cli.CliTest.testAddListRemove` is failing unrelated to the changes I was working on https://github.com/quarkusio/quarkus/pull/11852

https://github.com/quarkusio/quarkus/runs/1066539001

**Expected behavior**

The test to pass. 

**Actual behavior**

```log
[ERROR] Tests run: 7, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 169.318 s <<< FAILURE! - in io.quarkus.cli.CliTest
[ERROR] io.quarkus.cli.CliTest.testAddListRemove  Time elapsed: 1.703 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <quarkus-resteasy> but was: <>
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1124)
	at io.quarkus.cli.CliTest.testAddListRemove(CliTest.java:90)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:686)
```

--- a/devtools/platform-descriptor-json/src/main/java/io/quarkus/platform/descriptor/loader/json/impl/QuarkusJsonPlatformDescriptor.java
+++ b/devtools/platform-descriptor-json/src/main/java/io/quarkus/platform/descriptor/loader/json/impl/QuarkusJsonPlatformDescriptor.java
@@ -73,7 +73,7 @@ public class QuarkusJsonPlatformDescriptor implements QuarkusPlatformDescriptor,
     }
 
     private MessageWriter getLog() {
-        return log == null ? log = MessageWriter.DEFAULT : log;
+        return log == null ? log = MessageWriter.info() : log;
     }
 
     @Override

--- a/independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/CodestartInputBuilder.java
+++ b/independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/CodestartInputBuilder.java
@@ -14,7 +14,7 @@ public class CodestartInputBuilder {
     Collection<AppArtifactKey> dependencies = new ArrayList<>();
     Collection<String> codestarts = new ArrayList<>();
     Map<String, Object> data = new HashMap<>();
-    MessageWriter messageWriter = MessageWriter.DEFAULT;
+    MessageWriter messageWriter = MessageWriter.info();
 
     CodestartInputBuilder(CodestartResourceLoader resourceLoader) {
         this.resourceLoader = resourceLoader;

--- a/independent-projects/tools/codestarts/src/test/java/io/quarkus/devtools/codestarts/CodestartProcessorTest.java
+++ b/independent-projects/tools/codestarts/src/test/java/io/quarkus/devtools/codestarts/CodestartProcessorTest.java
@@ -20,7 +20,7 @@ class CodestartProcessorTest {
         spec.put("*", "replace");
 
         final CodestartProcessor processor = new CodestartProcessor(
-                MessageWriter.DEFAULT,
+                MessageWriter.info(),
                 Mockito.mock(CodestartResourceLoader.class),
                 "a",
                 Paths.get("test"),
@@ -37,7 +37,7 @@ class CodestartProcessorTest {
         spec.put("test/foo.tt", "forbidden");
 
         final CodestartProcessor processor = new CodestartProcessor(
-                MessageWriter.DEFAULT,
+                MessageWriter.info(),
                 Mockito.mock(CodestartResourceLoader.class),
                 "a",
                 Paths.get("test"),

--- a/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/data/QuarkusCommandInvocation.java
+++ b/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/data/QuarkusCommandInvocation.java
@@ -18,7 +18,7 @@ public final class QuarkusCommandInvocation extends ValueMap<QuarkusCommandInvoc
     }
 
     public QuarkusCommandInvocation(final QuarkusProject quarkusProject, final Map<String, Object> values) {
-        this(quarkusProject, values, MessageWriter.DEFAULT);
+        this(quarkusProject, values, MessageWriter.info());
     }
 
     public QuarkusCommandInvocation(final QuarkusProject quarkusProject, final Map<String, Object> values,

--- a/independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/descriptor/loader/json/QuarkusJsonPlatformDescriptorLoaderContext.java
+++ b/independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/descriptor/loader/json/QuarkusJsonPlatformDescriptorLoaderContext.java
@@ -12,7 +12,7 @@ public abstract class QuarkusJsonPlatformDescriptorLoaderContext implements Quar
     protected final ResourceLoader resourceLoader;
 
     public QuarkusJsonPlatformDescriptorLoaderContext(ArtifactResolver artifactResolver) {
-        this(artifactResolver, MessageWriter.DEFAULT);
+        this(artifactResolver, MessageWriter.info());
     }
 
     public QuarkusJsonPlatformDescriptorLoaderContext(ArtifactResolver artifactResolver, MessageWriter log) {

--- a/independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/tools/config/QuarkusPlatformConfig.java
+++ b/independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/tools/config/QuarkusPlatformConfig.java
@@ -31,7 +31,7 @@ public class QuarkusPlatformConfig {
         }
 
         private MessageWriter getMessageWriter() {
-            return log == null ? log = MessageWriter.DEFAULT : log;
+            return log == null ? log = MessageWriter.info() : log;
         }
 
         public Builder setPlatformDescriptor(QuarkusPlatformDescriptor platformDescr) {

--- a/independent-projects/tools/message-writer/src/main/java/io/quarkus/devtools/messagewriter/MessageWriter.java
+++ b/independent-projects/tools/message-writer/src/main/java/io/quarkus/devtools/messagewriter/MessageWriter.java
@@ -4,8 +4,6 @@ import java.io.PrintStream;
 
 public interface MessageWriter {
 
-    MessageWriter DEFAULT = info();
-
     static MessageWriter info() {
         return new DefaultMessageWriter();
     }

--- a/independent-projects/tools/platform-descriptor-resolver-json/src/main/java/io/quarkus/platform/descriptor/resolver/json/QuarkusJsonPlatformDescriptorResolver.java
+++ b/independent-projects/tools/platform-descriptor-resolver-json/src/main/java/io/quarkus/platform/descriptor/resolver/json/QuarkusJsonPlatformDescriptorResolver.java
@@ -176,7 +176,7 @@ public class QuarkusJsonPlatformDescriptorResolver {
 
     private void ensureLoggerInitialized() {
         if (log == null) {
-            log = MessageWriter.DEFAULT;
+            log = MessageWriter.info();
         }
     }
 

--- a/independent-projects/tools/platform-descriptor-resolver-json/src/test/java/io/quarkus/platform/descriptor/resolver/json/test/QuarkusJsonPlatformDescriptorResolverTest.java
+++ b/independent-projects/tools/platform-descriptor-resolver-json/src/test/java/io/quarkus/platform/descriptor/resolver/json/test/QuarkusJsonPlatformDescriptorResolverTest.java
@@ -25,7 +25,7 @@ public class QuarkusJsonPlatformDescriptorResolverTest extends ResolverSetupClea
 
     private static Path testDir;
 
-    private MessageWriter log = MessageWriter.DEFAULT;
+    private MessageWriter log = MessageWriter.info();
 
     @BeforeEach
     @Override

